js/app/components/mobile/ui.tsx

```diff
@@ -260,12 +262,12 @@ export function MobileUi() {
       {!isSelfAndNotLive && (
         <MobileChatPanel isPlayerRatioGreater={isPlayerRatioGreater} />
       )}
-      {muteWasForced && (
+      {muted && (
```

Comment:
Won't this put the big symbol over the stream even if I just muted it manually?

Reply:
Gotcha - happy to ship for now but let's open a ticket to get mute and fullscreen buttons back into the mobile UI? Got someone asking about it on a portrait desktop monitor too